docs/data-sources/gaussdb_opengauss_plugins.md

```diff
@@ -0,0 +1,32 @@
+---
+subcategory: "GaussDB"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_gaussdb_opengauss_plugins"
+description: |-
+  Use this data source to get list of supported plug-ins.
```

Comment:
Both `plugin` and `plug-in` are correct, but `plugin` (without the hyphen) is more commonly used in modern tech contexts.
```suggestion
  Use this data source to get list of supported plugins.
```